src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -18,6 +18,8 @@
   </ItemGroup>
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(OSGroup)' != 'AnyOS' AND '$(TargetGroup)' == 'netcoreapp'">
     <Compile Include="System\Data\SqlClient\PoolBlockingPeriod.cs" />
+    <Compile Include="System\Data\SqlClient\SqlBuffer.NetCoreApp.cs" />
```

Comment:
change to lowercase ```netcoreapp```

Reply:
why are we spliting the implementation into different files as netfx and netstandard dont use this implementation ?

Reply:
I followed the existing use of mixed case in the project, I can make it lower for this one but it won't be consistent throughout.

It was split apart at request of @AfsanehR in the original review on https://github.com/dotnet/corefx/pull/30371

Reply:
we are using the lowercase ```netcoreapp``` everywhere else in the repo when it is appearing as an extension.

Reply:
@Anipik The reason I used .NetCoreApp in #30371 is because we previously had [this file ](https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/ref/System.Data.SqlClient.NetCoreApp.cs) in SqlClient named as .NetCoreApp. Do you propose changing this file as well?

Reply:
We are pretty inconsistent about using the .netcoreapp.cs suffix for _implementation_ files that are conditionalized with `'$(TargetGroup)' == 'netcoreapp'`. The pattern is mainly for test files, since those are more often shared with .NET Framework. Personally I wouldn't use the suffix in implementation files but I don't think it matters although hopefully it's consistent within a project.
Casing should be lower case.